spec/factories/restaurant_factory.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                    :uuid             not null, primary key
+#  accept10bis           :boolean          default(FALSE), not null
+#  address               :string
+#  cuisine               :string
+#  latitude              :float
+#  longitude             :float
+#  max_delivery_time_min :integer
+#  name                  :string           not null
+#  rating                :float            default(0.0)
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#
+
+require 'faker'
+
+FactoryBot.define do
+  factory :restaurant, traits: [:name, :cuisine] do
```

Comment:
what about the other values? you should generate them all, as it will help you in the future.